server/utils/labels.ts

```diff
@@ -31,18 +31,19 @@ export function getFaderLabel(faderIndex: number, defaultName = 'CH'): string {
     const channelLabel = getChannelLabel(state, faderIndex)
 
     switch (state.settings[0].labelType) {

```

Comment:
This switch is duplicated in 'clint/utils/labels.ts'
Consider not having duplicated code.